### PR TITLE
dotCMS/core#23441 Fix spacing and match the design

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* UI Changes

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

image
image

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <div class="dot-container-code__content-tab-container">
-    <label class="block p-label-input-required field">{{
-        'message.containers.create.content_type_code' | dm
-    }}</label>
+    <div class="field mb-0">
+        <label class="block p-label-input-required">{{
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.html:3]: Where did you get this class `p-label-input-required`?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss [hunk 2]
```diff
@@ -56,10 +48,14 @@
 .dot-container-properties__code-loop-container {
     background: $white;
     border: 1px solid $input-border-color;
-    padding-bottom: 4rem;
 }
 
 :host ::ng-deep {
+    dot-api-link a {
+        width: $dot-api-link-width;
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss:55]: The problem I showed is not the width is the height, this is how it looks like in the content types:

image

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-loop-editor/dot-loop-editor.component.html [hunk 3]
```diff
@@ -8,13 +8,14 @@
             type="button"
         ></button>
     </div>
-    <div *ngIf="isEditorVisible">
-        <label class="block field" data-testId="label">{{
+    <div class="field mb-0" *ngIf="isEditorVisible" [@enterAnimation]>
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-loop-editor/dot-loop-editor.component.html:11]: Nice touch!

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 @use "variables" as *;
 
 .dot-container-code__content-tab-container {
-    padding: $spacing-4;
+    padding: 0 $dot-container-spacing 0 $dot-container-spacing;
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:4]: Use the closing spacing variable to this... sorry the design didn't match any of the spacing variables.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.html (hunk 1 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.html:3]: It is getting from the _misc.scss file.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss (hunk 4 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:4]: Noted

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss (hunk 2 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss:55]: Noted

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss (hunk 4 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss]: updated

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss (hunk 2 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.scss]: Updated

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss [hunk 5]
```diff
@@ -1,7 +1,7 @@
 @use "variables" as *;
 
 .dot-container-code__content-tab-container {
-    padding: $spacing-4;
+    padding: 0 $spacing-4 0 $spacing-4;
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.scss:4]: short hand: `padding: 0 $spacing-4`

**Review by alfredo-dotcms — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**